### PR TITLE
vendor/Makefile: page-to-alto is phony

### LINKED ISSUE #153: regression: page-to-alto is missing
Recently, we lost ability for `page alto` conversion. For some reason, during `make install`, no `page-to-alto` gets installed. Investigating...

### PR DESCRIPTION
**Pull request by bertsky**
fixes #153

### CODE REVIEW

**Review by kba — APPROVED** [review]

**Review by stweil — COMMENTED** [review]
I am not sure that this is the right fix (even if it seems to fix the problem). `gcv2hocr` is missing, too. And I also did not find a regression. Did the installation ever work before? If yes, with which version?

File: vendor/Makefile [hunk 1]
```diff
@@ -38,7 +38,7 @@ ALTO2PAGE_DIR = JPageConverter
 #     wget -O '$@' '$(SAXON_BROWSER_URL)'
 #}}}
 
-.PHONY: all check $(PAGE_SCHEMA_REPO) $(ABBYY_SCHEMA_REPO)
+.PHONY: all check $(PAGE_SCHEMA_REPO) $(ABBYY_SCHEMA_REPO) page-to-alto
```
Comment (stweil) [inline vendor/Makefile:41]:
```suggestion
.PHONY: all check $(PAGE_SCHEMA_REPO) $(ABBYY_SCHEMA_REPO) gcv2hocr page-to-alto
```

**Review by stweil — CHANGES_REQUESTED** [review]

**Comment by stweil** [comment]
In the old code the installation of `page-to-alto` was a side effect of `make all`. `make install` (which also respects `DESTDIR`) had no effect on its installation.

The suggested pull request restores the old behaviour partially. With my addition, it also restores `make` for `gcv2hocr` (which is built, but never installed).

**Comment by stweil** [comment]
`gcv2hocr` needs one more fix. Therefore I merge now and apply that fix on top.

**Pull request merged**